src/utils/personDataUtil.ts

```diff
@@ -6,6 +6,12 @@ export const skjermingskode = (person: PersonData) => {
     : '';
 };
 
+export const veilederEllerUfordelt = (person: PersonData) => {
+  return person.veileder
+    ? person.veileder.etternavn + ', ' + person.veileder.fornavn
```

Comment:
Dette kan skriver sånn: ``${person.veileder.etternavn}, ${person.veileder.fornavn}``
Jeg synes det blir finere enn `+`, men jeg er åpen for andre meninger.